navigation/navigation.ts

```diff
@@ -162,6 +162,9 @@ export const navigation = (locale: AppLocale): NavItemDefinition[] => [
       {
         slug: 'quick-start',
       },
+      {
+        slug: 'base',
```

Comment:
Wondering if the slug should be `base-testnet` since Pranav asked us to call it "testnet"

Reply:
Good call!